lib/appdev_support/pryrc/full.rb

```diff
@@ -0,0 +1,43 @@
+require "tabulo"
+Pry.config.print = proc do |output, value, _pry_|
```

Comment:
Is there some easy way to truncate the table cells to some maximum number of characters (idk, like 50)? I tested this with MSM queries and those `bio` columns are really long and render the table pretty un-readable:

<img width="960" alt="image" src="https://user-images.githubusercontent.com/12408296/235256060-6199473c-914b-45aa-9f5e-8266d9adf52e.png">

Reply:
Good catch! I dropped in the configuration from an old PR and didn't extensively test it. Cells are now also truncated.